jenkins_jobs/trigger_nightly_host_os_build/pre_build_script.sh

```diff
@@ -0,0 +1,3 @@
+RELEASE_DATE=$(date +%Y-%m-%d)
+echo "$RELEASE_DATE" > NIGHTLY_DIR_NAME
+echo "$TRIGGERED_BUILD_NUMBER_build-host-os"
```

Comment:
Right. It does nothing at this point. We have the same for trigger_weekly. I'll remove from both.